src/colorizer/types.ts

```diff
@@ -134,6 +134,18 @@ export type DrawSettings = {
   color: Color;
 };
 
+// CHANGING THESE VALUES CAN POTENTIALLY BREAK URLs. See `url_utils.parseDrawSettings` for parsing logic.
+export enum TabType {
+  FILTERS = "filters",
+  TRACK_PLOT = "track_plot",
+  SCATTER_PLOT = "scatter_plot",
+  SETTINGS = "settings",
+}
```

Comment:
There is a matching unit test that checks for these explicit string values, to make sure they don't accidentally get changed in the future.